scripts/deploy-azure-neon-proof.ps1

```
@@ -0,0 +1,136 @@
+    [Parameter(Mandatory = $true)]
+    [string]$DatabaseUrl,
+    [Parameter(Mandatory = $true)]
+    [string]$DatabaseUrlSync,
+    [string]$BifrostDocsSecretKey = "",
+    [switch]$SkipFrontendBuild,
+    [switch]$SkipInitJob
+)
+
+$ErrorActionPreference = "Stop"
+
+function Require-Command($Name) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Require-Command' uses an unapproved verb.

(PSUseApprovedVerbs)